web/frontend/src/components/base/index.svelte

```diff
@@ -202,10 +202,8 @@
   };
 
   const handleOnBlur = () => {
-    if (pressed.size <= 0) {
-      pressed.clear();
-      stop();
-    }
+    stop();
+    pressed.clear();
```

Comment:
Blur events should trigger both regardless of pressed.size

Reply:
So I am a little concerned about this, especially as I think [this comment](https://github.com/viamrobotics/rdk/pull/2943/files#diff-c968d5f79d59c182db75b543384c98a6d083bb7174295d1540602ca7e0021c7cR210) was for this `if` and it was moved away during refactoring. We checked against the pressed size to not interfere with scripts. So for example, If I ran a script to make a rover go in a square, than focus away from the RC to look at the logs, this would stop the script correct?

Reply:
Good point; let me do some more testing

Reply:
@DTCurrie @ethanlook after testing I am not sure this is needed at all

- I can't reproduce the bug in production; going to ask others to try
- Devin, for the logs example one would have to have two windows open, but I see what you're saying. (Clicking on the logs tab destroys the control component)

Reply:
My example is from a real world case that we added that check for.